lib/ansible/modules/remote_management/oneview/oneview_fcoe_network.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/python
+# Copyright (c) 2016-2017 Hewlett Packard Enterprise Development LP
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: oneview_fcoe_network
+short_description: Manage OneView FCoE Network resources.
+description:
+    - Provides an interface to manage FCoE Network resources. Can create, update, or delete.
+version_added: "2.4"
+requirements:
+    - "python >= 2.7.9"
+    - "hpOneView >= 4.0.0"
+author: "Felipe Bulsoni (@fgbulsoni)"
+options:
+    state:
+        description:
+            - Indicates the desired state for the FCoE Network resource.
+              C(present) will ensure data properties are compliant with OneView.
+              C(absent) will remove the resource from OneView, if it exists.
+        choices: ['present', 'absent']
+    data:
+        description:
+            - List with FCoE Network properties.
+        required: true
+
+extends_documentation_fragment:
+    - oneview
+    - oneview.validateetag
+'''
+
+EXAMPLES = '''
+- name: Ensure that FCoE Network is present using the default configuration
+  oneview_fcoe_network:
+    config: "{{ config_file_path }}"
+    state: present
+    data:
+      name: 'Test FCoE Network'
+      vlanId: '201'
```

Comment:
Nice catch, in this specific case it works as both a string or an integer but it is better to leave it as an integer for clarity.